src/object/cli_obj.c

```diff
@@ -1874,6 +1874,7 @@ obj_ec_recov_cb(tse_task_t *task, struct dc_object *obj,
 	uint32_t			 extra_flags, i;
 	int				 rc;
 
+	D_INIT_LIST_HEAD(&task_list);
```

Comment:
I don't see this used in between this line and the old line.   What is the fix here?

Reply:
Ah, nevermind, it's the error path.